docs/edgeql/functions/converters.rst

```diff
@@ -23,3 +23,24 @@ the functions below allow more options for such conversions.
 
         db> SELECT to_json('{"hello": "world"}')['hello'];
         {'world'}
+
+.. eql:function:: std::to_str(json: json, fmt: OPTIONAL str={}) -> str
+
+    :index: stringify dumps
+
+    Return string representation of the input value.
```

Comment:
"of the input value." -> "of the input JSON" ?

Reply:
Actually I meant it to be generic "input value". The reason being that this will be a polymorphic converter for a lot of different inputs. It happens to only have JSON for the moment.